modules/mig/main.tf

```diff
@@ -197,6 +200,9 @@ resource "google_compute_health_check" "http" {
     response     = var.health_check["response"]
     proxy_header = var.health_check["proxy_header"]
   }
+  log_config {
```

Comment:
Looks like this was added in 3.71 https://github.com/hashicorp/terraform-provider-google/blob/v3.90.1/CHANGELOG.md#3710-june-07-2021
Can we bump the provider version here https://github.com/terraform-google-modules/terraform-google-vm/blob/28c77b553d1f4462ef21fe55f4e4bfb26eb6d892/autogen/versions.tf.tmpl#L20-L22